examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -55,10 +50,5 @@ public void testE2EWordCount() throws Exception {
         options.getJobName(), "output", "results"}));
```

Comment:
You shouldn't need to extend DataflowPipelineOptions in WordCountITOptions anymore.